examples/gimbal/gimbal.cpp

```diff
@@ -85,81 +85,116 @@ int main(int argc, char** argv)
 
     std::cout << "Start controlling gimbal...\n";
     Gimbal::Result gimbal_result = gimbal.take_control(gimbal_id, Gimbal::ControlMode::Primary);
+    sleep_for(seconds(5));
```

Comment:
Is this sleep required?

Reply:
@julianoes No, the sleep is not required here. I've removed it.
Initially, I added it because I was encountering issues with taking control, and I thought introducing a delay might help. But it doesn't make sense as it is not an async method.